### PR TITLE
Update Highfreq Handler & Ops / Fix a Highfreq Bug

### PR DESCRIPTION
**Pull request by bxdd**
## Description

Update Highfreq Handler & Ops / Fix a Highfreq Bug
## How Has This Been Tested?
- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [x] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [x] Fix bugs
- [x] Add new feature
- [ ] Update documentation

### CODE REVIEW

File: examples/highfreq/highfreq_ops.py [hunk 1]
```diff
@@ -54,3 +54,25 @@ class IsNull(ElemOperator):
     def _load_internal(self, instrument, start_index, end_index, freq):
         series = self.feature.load(instrument, start_index, end_index, freq)
         return series.isnull()
+
+
+class Cut(ElemOperator):
+    def __init__(self, feature, l=None, r=None):
+        self.l = l
+        self.r = r
+        if (self.l != None and self.l <= 0) or (self.r != None and self.r >= 0):
```
Comment (you-n-g) [inline examples/highfreq/highfreq_ops.py:63]: `is None` is better than  `!= None`

File: examples/highfreq/highfreq_ops.py [hunk 2]
```diff
@@ -54,3 +54,25 @@ class IsNull(ElemOperator):
...
+
+        super(Cut, self).__init__(feature)
+
+    def _load_internal(self, instrument, start_index, end_index, freq):
+        series = self.feature.load(instrument, start_index, end_index, freq)
+        return series.iloc[self.l : self.r]
+
+    def get_extended_window_size(self):
+        ll = 0 if self.l == None else self.l
+        rr = 0 if self.r == None else abs(self.r)
```
Comment (you-n-g) [inline examples/highfreq/highfreq_ops.py:74]: Why abs is only used for `self.r` ?

File: examples/highfreq/highfreq_ops.py [hunk 3]
```diff
@@ -54,3 +54,25 @@ class IsNull(ElemOperator):
     def _load_internal(self, instrument, start_index, end_index, freq):
         series = self.feature.load(instrument, start_index, end_index, freq)
         return series.isnull()
+
+
+class Cut(ElemOperator):
+    def __init__(self, feature, l=None, r=None):
```
Comment (you-n-g) [inline examples/highfreq/highfreq_ops.py:60]: Add more documentation about it

File: examples/highfreq/highfreq_ops.py (hunk 2 above)
Comment (bxdd) [reply examples/highfreq/highfreq_ops.py:74]: I hope that self.l > 0 and self.r < 0. It mean cut slice `[self.l, self.len + r)` of raw data.

File: examples/highfreq/highfreq_ops.py (hunk 3 above)
Comment (bxdd) [reply examples/highfreq/highfreq_ops.py:60]: ok

File: examples/highfreq/highfreq_ops.py [hunk 4]
```diff
@@ -18,49 +32,147 @@ def get_calendar_day(freq="day", future=False):
...
+    feature:
+        A series with the first l and last -r elements deleted from the feature.
+        Note: It is deleted from the raw data, not the sliced data
+    """
+
     def __init__(self, feature, l=None, r=None):
         self.l = l
         self.r = r
-        if (self.l != None and self.l <= 0) or (self.r != None and self.r >= 0):
+        if (not self.l is None and self.l <= 0) or (not self.r is None and self.r >= 0):
```
Comment (you-n-g) [inline examples/highfreq/highfreq_ops.py:175]: The code is right, but it looks weird.
`XXX is not YYYY`  is preferred than `not  XXX is YYYY`

**Pull request merged**